lua/refactoring/tasks/apply_text_edits.lua

```diff
@@ -18,21 +11,17 @@ local function refactor_apply_text_edits(refactor)
     local edits = {}
 
     for _, edit in pairs(refactor.text_edits) do
-        local bufnr = edit.bufnr or refactor.buffers[1]
+        local bufnr = refactor.buffers[1]
```

Comment:
I believe this will mess with the "Extract Function to File" functionality (i.e. it will put the function into the same file rather than the new file). While testing this I also found various other errors with the "Extract Function to File" functionality due to small changes in recent commits that I didn't catch before, but I will make a  new PR to address these.

Reply:
You are right. I will fix this.